package.json

```diff
@@ -1,8 +1,14 @@
 {
   "name": "hubot-forecast",
   "description": "A hubot script to alert for inclement weather",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
I’ve no problem with the addition of yourself as a contributor, but I don’t bump version numbers until a release is cut.

Would you mind dropping this change? Esp. since this isn’t a bugfix, but a new feature (i.e., the middle digit will be what I’ll increment).